packages/search/src/Search.ts

```diff
@@ -98,15 +130,28 @@ export class Search extends Textfield {
         );
     }
 
+    private _manageClearButtonListeners(): void {
+        // add clear button listener when button is added to the DOM
+        if (this.clearButton && !this.clearButtonAbortController) {
+            this.clearButtonAbortController = new AbortController();
```

Comment:
Same here you are recreating this controller everytime this method reinstantiates.

Reply:
As mentioned previously I can't reuse the same controller. If it seems like a significant issue I can remove the `AbortController` and instead use `add/removeEventListener` API along with a flag that checks the listeners have been added to achieve the same result